src/index.tsx

```diff
@@ -1,16 +1,20 @@
 import React from 'react';
 import { createRoot } from 'react-dom/client';
 import { HashRouter } from 'react-router-dom';
+import { Provider } from 'react-redux';
 import { App } from './App';
+import { store } from './components/Slider/redux/store';
```

Comment:
I think const  store should be in folder store, in root of folder srs:
-srs(folder)
   - store(folder)
     - index.ts(file)
       - const store

Reply:
+ , waiting for our redux structure then i will fix it